# msg/CapabilityUpdate.msg
int32 robot_id
float32[] capabilities
float32[] degradation_mask  # 1 for degraded capability, 0 otherwise
bool in_recovery
int64 timestamp  # milliseconds since epoch